docs/major-mode.md

```diff
@@ -45,8 +45,7 @@ Required extensions:
 
 Documentation:
 
-- [VSpaceCode and Calva install guide](http://practicalli.github.io/clojure/clojure-editors/editor-install-guides/vspacecode-calva.html#install-vs-code-extension)
-- [VSpaceCode and Calva user guide](http://practicalli.github.io/clojure/clojure-editors/editor-user-guides/vspacecode-calva.html)
+- [Practicalli VSpaceCode](https://practical.li/vspacecode/) install & guide for VSpaceCode and Calva for Clojure development (updated Febrary 2023)
```

Comment:
```suggestion
- [Practicalli VSpaceCode](https://practical.li/vspacecode/) install & guide for VSpaceCode and Calva for Clojure development
```

I would suggest removing the updated, but other than that LGTM. Thanks for updating